cmd/group/main.go

```diff
@@ -38,7 +40,8 @@ func main() {
 
 	logLevel := len(log.AllLevels) - 2
 
-	listen := "unix:///run/infrakit/plugins/group.sock"
+	listen := "unix:///run/infrakit/plugins/"
+	sock := "group.sock"
```

Comment:
The idea behind it is that I can now set 1 listen dir for the entire app while still being able to declare the socket via the `--sock` flag. 
Otherwise the `--listen` flag needs to be over-written every time, since it includes the `group.sock` in its string